documentation/book/con-kafka-listeners.adoc

```diff
@@ -21,6 +21,7 @@ When none of these properties are defined, the listener will be disabled.
 listeners:
   plain: {}
   tls: {}
+  external: {}
```

Comment:
External on its own does not work. It needs additional type. E.g.

```yaml
  plain: {}
  tls: {}
  external:
    type: loadbalancer
```

Reply:
Thanks Jakub

Reply:
I pushed the update in.